src/helpers/order-process/order-process-helper.ts

```diff
@@ -184,20 +187,26 @@ export const addOrderProcess = async ({
   });
 
   const promiseB =
+    before_portfolio_item_id &&
+    before_portfolio_item_id !== '' &&
     before_portfolio_item_id !== undefined
       ? getOrderProcessApi().addOrderProcessBeforeItem(
           ((op as unknown) as OrderProcess).id as string,
           { portfolio_item_id: before_portfolio_item_id }
         )
       : {};
   const promiseA =
+    after_portfolio_item_id &&
+    after_portfolio_item_id !== '' &&
     after_portfolio_item_id !== undefined
       ? getOrderProcessApi().addOrderProcessAfterItem(
           ((op as unknown) as OrderProcess).id as string,
           { portfolio_item_id: after_portfolio_item_id }
         )
       : {};
   const promiseR =
+    return_portfolio_item_id &&
+    return_portfolio_item_id !== '' &&
     return_portfolio_item_id !== undefined
       ? getOrderProcessApi().addOrderProcessReturnItem(
           ((op as unknown) as OrderProcess).id as string,
```

Comment:
Feels like all of these could be refactored into a single function, but I don't know javascript conventions very well, so I don't know how easy/difficult it is to do that.